cmd/kubeadm/app/master/apiclient.go

```diff
@@ -219,3 +222,41 @@ func SetMasterNodeAffinity(meta *api.ObjectMeta) {
 	}
 	meta.Annotations[api.AffinityAnnotationKey] = string(affinityAnnotation)
 }
+
+func createDummyDeployment(client *clientset.Clientset) {
+	fmt.Println("<master/apiclient> attempting a test deployment")
+	dummyDeployment := NewDeployment("dummy", 1, api.PodSpec{
+		SecurityContext: &api.PodSecurityContext{HostNetwork: true},
+		Containers: []api.Container{{
+			Name:  "dummy",
+			Image: images.GetAddonImage("pause"),
+		}},
+	})
+
+	wait.PollInfinite(apiCallRetryInterval, func() (bool, error) {
```

Comment:
I'd like to find a way to improve our usage of `wait.PollInfinite()` everywhere, but haven't quite figured it out yet. We would definitely benefit from kubelet API, if it will allow us to do less polling and get instant confirmation of control plane components getting deployed or erroring otherwise...